### PR TITLE
Plane crud func

### PR DESCRIPTION
**Pull request by TheDoctorZee**

### CODE REVIEW

File: src/main/java/com/epam/lowcost/DAO/PlaneDAOImpl.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlaneDAOImpl implements PlaneDAO {
+
+    private DataSource dataSource;
```
Comment (EugeneGrim) [inline src/main/java/com/epam/lowcost/DAO/PlaneDAOImpl.java]: There are some changes in develop branch with datasource. Merge it into your branch and resolve conflicts

File: src/main/java/com/epam/lowcost/service/PlaneService.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+public interface PlaneService {
+
+    List<Plane> getAllPlanes();
+
+    Plane getById(long planeId);
+
+    Plane addPlane(Plane plane);
+
+    Plane updatePlane (Plane plane);
+}
```
Comment (EugeneGrim) [inline src/main/java/com/epam/lowcost/service/PlaneService.java]: It's not all the CRUD functionality

File: src/main/java/com/epam/lowcost/DAO/PlaneDAOImpl.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            if (update == 1) {
+                return plane;
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return plane;
+    }
+
+    private Plane getPlanesFromDB(ResultSet rs) throws SQLException {
```
Comment (vakhonin) [inline src/main/java/com/epam/lowcost/DAO/PlaneDAOImpl.java]: I think that this name is not appropriate. Maybe more suitable `extract...`... I'm sure you can create new good name)

File: src/main/java/com/epam/lowcost/DAO/PlaneDAOImpl.java [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        return Plane.builder()
+                .id(rs.getLong("id"))
+                .model(rs.getString("model"))
+                .businessPlacesNumber(rs.getInt("businessPlacesNumber"))
+                .economPlacesNumber(rs.getInt("economPlacesNumber"))
+                .build();
+    }
+}
```
Comment (vakhonin) [inline src/main/java/com/epam/lowcost/DAO/PlaneDAOImpl.java]: I did not find where you close connection

File: src/main/java/com/epam/lowcost/controller/PlaneController.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+@Controller
+@RequestMapping(value = "/plane")
+public class PlaneController {
+
+    @Autowired
+    PlaneService planeService;
+
+    @GetMapping(value = "/all")
+    public String listAllUsers(Model model) {
```
Comment (vakhonin) [inline src/main/java/com/epam/lowcost/controller/PlaneController.java]: list - it is noun. Method - need to be a verb

File: src/main/java/com/epam/lowcost/DAO/PlaneDAOImpl.java (hunk 1 above)
Comment (TheDoctorZee) [reply src/main/java/com/epam/lowcost/DAO/PlaneDAOImpl.java]: Merged and resolved

File: src/main/java/com/epam/lowcost/service/PlaneService.java (hunk 2 above)
Comment (TheDoctorZee) [reply src/main/java/com/epam/lowcost/service/PlaneService.java]: Delete method added

File: src/main/java/com/epam/lowcost/DAO/PlaneDAOImpl.java (hunk 3 above)
Comment (TheDoctorZee) [reply src/main/java/com/epam/lowcost/DAO/PlaneDAOImpl.java]: Changed method name

File: src/main/java/com/epam/lowcost/controller/PlaneController.java (hunk 5 above)
Comment (TheDoctorZee) [reply src/main/java/com/epam/lowcost/controller/PlaneController.java]: Changed method name

**Pull request merged**